src/js/rx/containers/Active.jsx

```diff
@@ -70,13 +69,17 @@ class Active extends React.Component {
   }
 
   renderViewSwitch() {
+    if (!this.props.prescriptions) {
+      return null;
+    }
+
     const toggles = [
       { key: 'card', value: 'Card' },
       { key: 'list', value: 'List' },
     ];
 
     return (
-      <div className="rx-view-toggle" ref={(elem) => { this.viewToggle = elem; }}>View:&nbsp;
+      <div className="rx-view-toggle show-for-small-only" ref={(elem) => { this.viewToggle = elem; }}>View:&nbsp;
```

Comment:
Doesn't this show up if it's in a mobile-sized window? Unless I'm confusing myself, I thought this toggle is supposed to appear only on desktop, since we wanted to always show the cards in mobile.